packages/sanity/src/structure/panes/document/statusBar/ActionMenuButton.tsx

```diff
@@ -16,16 +16,46 @@ export interface ActionMenuButtonProps {
   disabled: boolean
 }
 
-export function ActionMenuButton(props: ActionMenuButtonProps) {
-  const {actionStates, disabled} = props
-  const idPrefix = useId()
+/**
+ * @internal
+ */
+export function ActionDialogWrapper({
+  actionStates,
+  children,
+  referenceElement,
+}: {
+  actionStates: DocumentActionDescription[]
+  children: ({handleAction}: {handleAction: (idx: number) => void}) => React.ReactNode
```

Comment:
We should avoid usage of the default export from react (I thought eslint would catch this tbh)

```suggestion
  children: ({handleAction}: {handleAction: (idx: number) => void}) => ReactNode

```